packages/lite-insight/src/algorithms/change-point/bayesian.ts

```diff
@@ -0,0 +1,38 @@
+import BayesianChangePoint, { BreakPoint } from 'bayesian-changepoint';
```

Comment:
Should we make words format consistent?

`changepoint`, `breakpoint` or `change point`, `break point`. Here, it seems to be mixed.

Reply:
yes, we use `changepoint` in our code and doc